carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/resources/L10N.java

```diff
@@ -55,12 +55,16 @@ public class L10N {
     private static ArrayList<ResourceBundle> resBoundles = new ArrayList<ResourceBundle>();
     private static Properties missedResources = new Properties();
     
-    private static SoftAssert mistakes = new SoftAssert();
+    private static SoftAssert mistakes;
+
 
     /**
-     * Load L10N resource bundle.
+     * Load L10N resource bundle corresponding to a specific locale.
+     * If called setLocale function in the test, must be called to reload resources
      */    
     public static void load() {
+        // #1679 fix
```

Comment:
replace everywhere to `// #1679: L10N: made assertion threads dependent`

Reply:
done